test/local/log_tests.js

```diff
@@ -184,7 +186,7 @@ test(
       t.equal(metricsContext.gather.callCount, 1, 'metricsContext.gather was called once')
       t.equal(metricsContext.gather.args[0][0].event, 'foo', 'event property was correct')
 
-      t.equal(logger.info.callCount, 1, 'logger.info was called once')
+      t.equal(logger.info.callCount, 2, 'logger.info was called twice')
       t.equal(logger.info.args[0][1].service, 'bar', 'service property was correct')
```

Comment:
I kind of feel like if a property is being tested at this point in the `activityEvent` assertions, it should also be tested in the `flowEvent` assertions.